### ISSUE TITLE
Add missing TinyMCE localization files for various languages

### ISSUE THREAD

**Issue by softwaymedical** [tinymce/tinymce-dist#8]
The localizations can be downloaded on http://www.tinymce.com/i18n/index.php.

But this would be easier if the localizations were directly in this repository.

**Comment by althaus** [comment]
:+1:

**Comment by thet** [comment]
at least they should be included in tinymce-dist. see: https://github.com/tinymce/tinymce-dist/issues/4

**Comment by atremba** [comment]
Agree. @spocke would you please add languages to the repository or provide other convenient way to distributed translations alternative to manual downloading and copying?

This is actual for deployable systems with configurable locales or interfaces, e.g. one setup may use four languages while other setup only two.

**Comment by iquito** [comment]
I tried using tinymce via composer & packagist, but without the language files it just silently fails when defining a language. Adding the languages manually defeats the whole purpose of using composer to include tinymce. This detail makes this respository unusable in my opinion.

**Comment by michallohnisky** [comment]
:+1:

**Comment by Rvanlaak** [comment]
:+1:
